build.rs

```diff
@@ -0,0 +1,44 @@
+use std::env;
+use std::fs::File;
+use std::path::PathBuf;
+
+fn main() {
+    let nrf_wifi_path = PathBuf::from("third_party/nrf70-bm/nrf_wifi")
+        .canonicalize()
+        .expect("Cannot canonicalize path");
```

Comment:
This canonicalize prevents compilation on Windows.
See the note here: https://doc.rust-lang.org/std/fs/fn.canonicalize.html

If removed, it compiles fine. On linux it compiles as well, so it's not really necessary.
```suggestion
    let nrf_wifi_path = PathBuf::from("third_party/nrf70-bm/nrf_wifi");
```